test/Ocelot.AcceptanceTests/ConsulWebSocketTests.cs

```diff
@@ -126,7 +126,7 @@ private void GivenThereIsAFakeConsulServiceDiscoveryProvider(string url, string
                 if (context.Request.Path.Value == $"/v1/health/service/{serviceName}")
                 {
                     var json = JsonConvert.SerializeObject(_serviceEntries);
-                    context.Response.Headers.Add("Content-Type", "application/json");
+                    context.Response.Headers.Append("Content-Type", "application/json");

```

Comment:
Could you remove all these changes from the PR please?
I don't see any benefits, because it is out of the scope of work.

Reply:
[it's now a warning in .net8, asp0019](https://learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0019?view=aspnetcore-8.0)

Cause
IDictionary.Add isn't recommended for setting or appending headers. [IDictionary.Add](https://learn.microsoft.com/en-us/dotnet/api/system.collections.idictionary.add) throws an ArgumentException when attempting to add a duplicate key.

Rule description
IDictionary.Add isn't recommended for setting or appending headers.

How to fix violations
To fix a violation of this rule, use IHeaderDictionary.Append or the indexer to append or set headers.

Reply:
🆗 Not an issue